buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -446,27 +450,31 @@ public long numSmallAllocations() {
     }
 
     @Override
-    public long numNormalAllocations() {
+    public synchronized long numNormalAllocations() {
         return allocationsNormal;
     }
 
     @Override
     public long numDeallocations() {
-        return deallocationsTiny + deallocationsSmall + allocationsNormal + deallocationsHuge.value();
+        final long deallocs;
+        synchronized (this) {
+            deallocs = deallocationsTiny + deallocationsSmall + allocationsNormal;
```

Comment:
@normanmaurer 
Is this a bug?

deallocs = deallocationsTiny + deallocationsSmall + **de**allocationsNormal;